### DISCUSSION TITLE
Integration of National Map with Australian Scalable Drone Cloud platform

### DISCUSSION THREAD

**Discussion by TimeScience** [TerriaJS/terriajs#5624] [General] [answered]
Hi all,

I manage the ANU node of the Australian Plant Phenomics Facility (APPF). We are project partners on the Australian Scalable Drone Cloud [http://asdc.io/](http://asdc.io/) platform, a 3-year project funded by the NCRIS Australian Research Data Commons (ARDC). Quite the mouthful of acronyms!
The goal of the ASDC project is to build national drone platform to support drone-based research in Australia. The ARDC platforms projects are intended to support integration existing capacity into a national drone platform so we're using Open Drone Map for the photogrammetry, will probably use CKAN for the data and user management, potree for VR interactions with point clouds, etc. For downstream analytics and pipeline development, we are probably going to integrate with EcoCommons.

The Main scope of APPF's deliverables on the ASDC project are to support development of tools for data visualisation and interactivity. We've supported the addition of native VR capabilities into Potree and are also developing AR tools for interacting with time-series point cloud data. 
For data interactivity and mapping, Cesium/National Map seem like an obvious choice for this, particularly now that Cesium [natively integrates with Unreal Engine](https://cesium.com/platform/cesium-for-unreal/). 
Given all that, rather than us hiring a developer to do the National Map integration, I'm wondering if there is possible scope for us to start a collaborative project with terriajs staff or National Map people, to implement this integration. Any thoughts or suggestions appreciated, or DM or email me.

Cheers

Tim

**Comment by AnaBelgun** [comment, marked as answer]
hi @TimeScience, I replied by email.
